src/components/ChatApp/MessageComposer.react.js

```diff
@@ -476,6 +487,8 @@ class MessageComposer extends Component {
             autoFocus={focus}
             background={textarea}
             color={textcolor}
+            onClick={this.handleClick}
+            key={key}
```

Comment:
Is this key required? If yes, Please explain why it is being used.

Reply:
@fragm Yes when we click outside of message composer the text area becomes out of focus and clicking again it does not focus again. So I have to update key state on clicking inside the text area so that it is focused again.